react/private/withTextProps.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import omit from 'lodash/omit';
+import forEach from 'lodash/forEach';
+
+export const sizes = [
+  'small',
+  'substandard',
+  'superstandard',
+  'standard',
+  'subheading',
+  'headline',
+  'heading',
+  'hero'
+];
+
+const getBooleanSizes = () => {
+  const booleanProps = {};
+
+  forEach(sizes, size => {
+    booleanProps[size] = PropTypes.bool;
+  });
```

Comment:
We should be returning `booleanProps` here after the `forEach`

Reply:
Fixed